_layouts/default.html

```diff
@@ -89,6 +89,12 @@
     </div>
     <div class="flex max-width">
         <div class="defaultwrapper" id="defaultwrapper">
+        <input type="checkbox" class="openSidebarMenu" id="openSidebarMenu" />
```

Comment:
Several points:

* (Does this really need a checkbox?)
* Is the code valid?
* Please don’t “close” void elements (let’s make this a rule).